File: LLM-Translations/GPT-4/avatar/Python/Go/codeforces_99_A.go
package main

import (
	"fmt"
	"strconv"
	"strings"
)

func main() {
	var l string
	fmt.Scanln(&l)
	s := strings.Split(l, ".")
	p := s[1]
	i, _ := strconv.Atoi(string(p[0]))

	if s[0][len(s[0])-1] == '9' {
		fmt.Println("GOTO Vasilisa.")
	} else if s[0][len(s[0])-1] != '9' && i < 5 {
		fmt.Println(s[0])
	} else {
		num, _ := strconv.Atoi(s[0])
		num++
		fmt.Println(strconv.Itoa(num))
	}
}

// End of Code

File: LLM-Translations/GPT-4/avatar/Python/C/codeforces_99_A.c

#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() {
    char l[100];
    fgets(l, 100, stdin);
    char *s = strtok(l, ".");
    char *p = strtok(NULL, ".");
    int i = p[0] - '0';

    int last_digit = s[strlen(s) - 1] - '0';

    if (last_digit == 9) {
        printf("GOTO Vasilisa.\n");
    } else if (last_digit != 9 && i < 5) {
        printf("%s\n", s);
    } else {
        int incremented = atoi(s) + 1;
        printf("%d\n", incremented);
    }

    return 0;
}
// End of Code


